tests/cloudpickle_test.py

```diff
@@ -1533,7 +1533,13 @@ def process_data():
             # grown by more than a few MB as closures are garbage collected at
             # the end of each remote function call.
             growth = w.memsize() - reference_size
-            assert growth < 1e7, growth
+
+            # Note that an "empty" Python interpreter on MacOS uses a
+            # significantly bigger amount of memory than on Linux systems
+            # (~10MB vs ~1MB), so the upper-bound on the memory use growth used
+            # below is only tight on Mac, and could be largely reduced on
+            # Linux.
```

Comment:
It's not the "empty interpreter". The empty interpreter size is at least 30MB if you import a couple of packages, including on Linux.

What we measure here is the memory growth when running this loop.

Maybe just mentions that that for some reason the growth on macOS is around 10MB while it's only 1MB on Linux but in both cases it does normally stay bounded when we increase the number of iterations and in particular it stays significantly lower than the 100MB one my expect in case of a reference leak.